src/api_keys/api.rs

```diff
@@ -78,16 +79,27 @@ pub async fn make_request(
     let url = user_make_request_info
         .api_url
         .replace("xxxREPLACE_MExxx", &api_key_info);
+
+    let mut headers = HeaderMap::new();
+    for (key, value) in &user_make_request_info.http_headers {
+        let first = key.replace("xxxREPLACE_MExxx", &api_key_info);
+        let second = value.replace("xxxREPLACE_MExxx", &api_key_info);
+
+        let header_name = HeaderName::from_bytes(first.as_bytes()).unwrap();
```

Comment:
I think its unlikely this would ever panic, but i think we should either handle the error anyway or use `expect` as a way of showing intentionally wanted to panic here rather than just forgot to remove an unwrap.

Reply:
nope this was just a mistake will fix